### DISCUSSION TITLE
@threads  :dynamic  expr

### DISCUSSION THREAD

**Discussion by serpenttom** [JuliaLang/julia#46301] [General]
There is an ## Example for @threads in  threadingconstructs.jl.   My Julia is v1.7.3.  The first example,  @threads  :static expr runs in success. 

 _6.021147 seconds (62.77 k allocations: 3.639 MiB, 1.13% compilation time)_

However, the second example fails.

_ERROR: LoadError: ArgumentError: unsupported schedule argument in @threads
Stacktrace:
 [1] var"@threads"(__source__::LineNumberNode, __module__::Module, args::Vararg{Any})
   @ Base.Threads .\threadingconstructs.jl:137
in expression starting at REPL[4]:3_

How can I solve this ERROR.

Thank you very much.

**Comment by serpenttom** [comment]
I have tried in v1.8.0. and both example succeed.  It may be owing to the version of Julia.

**Comment by LilithHafner** [comment]
I'm glad you figured it out! `:dynamic` threads are a feature that is not present in 1.7 or earlier versions of Julia.

For future reference, this sort of question is best directed to the #hepdesk [slack](https://julialang.org/slack/) channel or the Julia [discourse](https://discourse.julialang.org/).
